interface/forms/clinical_notes/new.php

```diff
@@ -15,7 +15,6 @@
  * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
  */
 
-$ignoreAuth = 1;
```

Comment:
@sjpadgett , just making sure ok to remove this

Reply:
yep forgot from a FF debug.